fore/foresight/client.py

```diff
@@ -87,17 +87,17 @@ def create_simple_evalset(
             reference_answer = None
             if reference_answers:
                 reference_answer = reference_answers[i]
-            entries.append(
-                EvalsetEntry(query=query,
-                             reference_answer=reference_answer,
-                             entry_id=str(uuid.uuid4())))
+            entries.append(EvalsetEntry(query=query,
+                                        entry_id=str(uuid.uuid4())))
+            if reference_answer:
+                entries[-1].reference_answer = reference_answer
```

Comment:
```suggestion
            new_entry = EvalsetEntry(query=query,
                                     entry_id=str(uuid.uuid4())))
            if reference_answer:
                new_entry.reference_answer = reference_answer
            entries.append(new_entry)
```
I believe it is a bit easier to read this way

Reply:
and I share your beliefs!